aptos-move/aptos-vm-types/src/tests/test_change_set.rs

```diff
@@ -740,24 +752,49 @@ mod tests {
 
     macro_rules! assert_group_write_size {
         ($op:expr, $s:expr, $exp:expr) => {{
-            let group_write = GroupWrite::new($op, BTreeMap::new(), $s, $s);
+            let group_write = GroupWrite::new($op, BTreeMap::new(), $s, $s.get());
             assert_eq!(group_write.maybe_group_op_size(), $exp);
         }};
     }
 
     #[test]
     fn test_group_write_size() {
         // Deletions should lead to size 0.
-        assert_group_write_size!(WriteOp::legacy_deletion(), 0, None);
+        assert_group_write_size!(
+            WriteOp::legacy_deletion(),
+            ResourceGroupSize::zero_combined(),
+            None
+        );
         assert_group_write_size!(
             WriteOp::Deletion {
                 metadata: raw_metadata(10)
             },
-            0,
+            ResourceGroupSize::zero_combined(),
             None
         );
 
-        let sizes = [20, 100, 45279432, 5];
+        let sizes = [
+            ResourceGroupSize::Combined {
+                num_tagged_resources: 1,
+                all_tagged_resources_size: 20,
+            },
+            ResourceGroupSize::Combined {
+                num_tagged_resources: 1,
+                all_tagged_resources_size: 100,
+            },
+            ResourceGroupSize::Combined {
+                num_tagged_resources: 1,
+                all_tagged_resources_size: 45279432,
+            },
+            ResourceGroupSize::Combined {
+                num_tagged_resources: 1,
+                all_tagged_resources_size: 5,
+            },
+            ResourceGroupSize::Combined {
```

Comment:
Before we had 4 sizes, now it's 5? The last one is not needed/tested?

Reply:
I added one more test now, that num_tagged_resources is separated? :)